django/core/cache/backends/dummy.py

```diff
@@ -12,29 +12,50 @@ def add(self, key, value, timeout=DEFAULT_TIMEOUT, version=None):
         self.validate_key(key)
         return True
 
+    async def aadd(self, *args, **kwargs):
```

Comment:
Dummy backend no longer needs new methods after changing `BaseCache` implementation as proposed in the [comment](https://github.com/django/django/pull/13547/files#r637837628). Please revert changes in the `DummyCache`.

Reply:
Seems inefficient though, no?